app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,39 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
```

Comment:
@niranjan94 I did not understand the looping part.  For now, we are having two nested loops, the outer one is the subtopics which are returned by the controller(according to the topics) and the inner one where we are looping the model and checking whther the both of the subTopics match or not.
Isn't this a correct approach? I mean can you explain me what is needed there?